gpt4all-chat/embllm.h

```diff
@@ -50,6 +51,9 @@ public Q_SLOTS:
 private Q_SLOTS:
     void handleFinished();
 
+protected:
```

Comment:
Why is this protected? Am I missing that you're subclassing it?

Reply:
I can make it private. Python has put me in the habit of erring on the side of greater visibility.